static/js/tools/timeline/page.tsx

```diff
@@ -125,6 +129,40 @@ class Page extends Component<unknown, PageStateType> {
       </>
     );
   }
+
+  private addPlaceAction(place: string): void {
+    // We only need to check the availability of selected stat vars when adding
```

Comment:
nit: more explicit "as we take the union of the eligible stat vars for all places"

Reply:
done